### ISSUE TITLE
Update advocacy.mozilla.org README

### ISSUE THREAD

**Issue by gideonthomas** [MozillaFoundation/Advocacy#46]
A bunch of new changes have been made to how the code is arranged and run and this isn't reflected in the repo's readme file. We should update it.

**Comment by damiankay** [comment]
Hey @gideonthomas - who would be responsible for making these updates?

**Comment by gideonthomas** [comment]
cc @ScottDowne

**Comment by ScottDowne** [comment]
Yup this would prob fall under me as something I can do during down time.
